commons/commons-serialization-util/src/main/kotlin/uk/ac/ebi/serialization/common/FilesResolver.kt

```diff
@@ -15,23 +11,14 @@ class FilesResolver(private val basePath: File) {
         fileName: String,
     ): File {
         val name = fileName.replace("/", "-")
-        return baseFolder(subAccNo, version.toString()).createTempFile(name, ".json")
+        val baseFolder = basePath.resolve("$subAccNo/$version")
+        baseFolder.mkdirs()
+
+        return baseFolder.createTempFile(name, ".json")
```

Comment:
what was the problem with previous base folder calculation? why remove date?

Reply:
1. it's easier to identify which files belong to a specific submission request based on the accNo/version combination
2. it's the same approach we're using for temporary fire files generation (such as zip)
3. since we're using the default temp file creation mechanism, they will still have a unique id

Reply:
maybe reanme function to create request tmp file?